proto/recon.proto

```diff
@@ -148,15 +142,21 @@ message RecognizePlacesResponse {
 
 // RecogPlace contains info of a place for RecognizePlaces method.
 message RecogPlace {
-  message Name {
-    repeated string parts = 1;
-  }
+  message Name { repeated string parts = 1; }
   repeated Name names = 1;
   string dcid = 2;
   repeated string containing_places = 3;
   int64 population = 4;
 }
 
-message RecogPlaces {
-  repeated RecogPlace places = 1;
-}
+message RecogPlaces { repeated RecogPlace places = 1; }
+
+message TokenSpans {
```

Comment:
This is for internal use right?  For my understanding, curious why this is a proto instead of a struct?

Reply:
Right, this is to make the comparisons in unit test easier: 

```
cmpOpts := cmp.Options{
  protocmp.Transform(),
}
if diff := cmp.Diff(got, c.want, cmpOpts); diff != "" {
  ...
}
```